### ISSUE TITLE
Add support for mapping nested values

### ISSUE THREAD

**Issue by dominik-hadl** [nodes-ios/Serializable#153]
This might be useful in cases where we don't need to parse a whole nested object and just care about one value.

For example:

JSON
```
{
    "name": "Your name",
    "times": {
        "createdAt": "some time",
        "updatedAt": "some time",
        "deletedAt": "some time"
    }
}
```

Model
```
struct User {
    var name = ""
    var createdAt = Date() // <-times/createdAt
}
```

**Comment by jakobmygind** [comment]
I have implemented this. Needs review.

**Comment by dominik-hadl** [comment]
Implemented in #156.
